From: hololeap <hololeap@protonmail.com>
Signed-off-by: hololeap <hololeap@protonmail.com>

The `build-depends` in the original .cabal file is incorrect and
needs to be modified.

--- tasty-discover-4.2.2-new/tasty-discover.cabal
+++ tasty-discover-4.2.2-new2/tasty-discover.cabal
@@ -69,20 +69,20 @@
 executable              tasty-discover
   if !flag(executable)
     buildable: False
-  import:               base
+  build-depends:        base
                       , Glob
                       , containers
                       , directory
                       , filepath
+                      , tasty-discover
   main-is:              executable/Main.hs
   autogen-modules:      Paths_tasty_discover
   other-modules:        Paths_tasty_discover
   ghc-options:          -Wall
-  build-depends:        tasty-discover
   default-language:     Haskell2010
 
 test-suite tasty-discover-test
-  import:               base
+  build-depends:        base
                       , Glob
                       , containers
                       , directory
@@ -94,6 +94,8 @@
                       , tasty-hunit
                       , tasty-quickcheck
                       , tasty-smallcheck
+                      , tasty-discover
+                      , hspec
   type:                 exitcode-stdio-1.0
   main-is:              Driver.hs
   ghc-options:          -threaded -rtsopts -with-rtsopts=-N
@@ -106,6 +108,5 @@
   autogen-modules:      Paths_tasty_discover
   hs-source-dirs:       test
   ghc-options:          -Wall
-  build-depends:        tasty-discover
   default-language:     Haskell2010
   build-tool-depends:   tasty-discover:tasty-discover
